UnityProject/Assets/Scripts/Core/Chat/EmoteActionManager.cs

```diff
@@ -7,14 +7,19 @@ namespace Core.Chat
 {
 	public class EmoteActionManager : MonoBehaviour
 	{
-		[SerializeField]
-		private List<EmoteSO> emotes;
+		public static EmoteActionManager Instance;
```

Comment:
if this is going to be a singleton monobehaviour, why not use the class we made for them?

Reply:
Rider kept bugging me about it for some reason so I just gave up and did it manually

Reply:
What was it complaining about?

Reply:
> What was it complaining about?

Something related to a manager namespace? idk i kept trying to find what it wanted but even the quick actions didn't give me anything

Reply:
rider finally pissed off